vendor/assets/javascripts/ckeditor/ckeditor.js

```diff
@@ -691,8 +691,8 @@ CKEDITOR.plugins.add("menubutton",{requires:"button,menu",onLoad:function(){var
 c.on=0},this);this.onMenu&&e.addListener(this.onMenu)}this.setState(CKEDITOR.TRISTATE_ON);c.on=1;setTimeout(function(){e.show(CKEDITOR.document.getById(c.id),4)},0)}};CKEDITOR.ui.menuButton=CKEDITOR.tools.createClass({base:CKEDITOR.ui.button,$:function(g){delete g.panel;this.base(g);this.hasArrow=true;this.click=a},statics:{handler:{create:function(a){return new CKEDITOR.ui.menuButton(a)}}}})},beforeInit:function(a){a.ui.addHandler(CKEDITOR.UI_MENUBUTTON,CKEDITOR.ui.menuButton.handler)}}),CKEDITOR.UI_MENUBUTTON=
 "menubutton",CKEDITOR.plugins.add("fontsize",{requires:"menubutton",init:function(a){function g(a,b,c){this.editor=a;this.name=b;this.value=c;this.context="p"}g.prototype.exec=function(a){var b=a.getSelection();if(b){var c=b.createBookmarks();if(this.editor.config.advancedEditor){var f=this,e=function(a){var b=a.getName(),c=a.getParent(),d=c.getName(),d=c.getAttribute("contenteditable")&&d==="div"&&b!=="ul"&&b!=="ol";return b==="p"||b==="li"||d?a:e(c)};(b.getRanges()||[]).map(function(a){for(var b=
 new CKEDITOR.dom.walker(a),a=b.next()||a.endPath().elements[0];a;){a.type===CKEDITOR.NODE_TEXT&&(a=a.getParent());a=e(a);f.value&&a.setStyle("font-size",f.value+"%");a=b.next()}})}else{var g=a.createRange();g.selectNodeContents(a.editable());g=g.createIterator();for(g.enlargeBr=true;block=g.getNextParagraph("p");)if(!block.isReadOnly()){block.removeStyle("font-size");this.value!==100&&block.setStyle("font-size",this.value+"%")}}a.focus();a.forceNextSelectionCheck();b.selectBookmarks(c)}};for(var c=
-[{label:a.lang.fontsize.smaller,name:"fontsizeSmaller",size:60},{label:a.lang.fontsize.small,name:"fontsizeSmall",size:80},{label:a.lang.fontsize.normal,name:"fontsizeNormal",size:100},{label:a.lang.fontsize.large,name:"fontsizeLarge",size:130},{label:a.lang.fontsize.larger,name:"fontsizeLarger",size:160}],e=0;e<c.length;e++){var i=c[e],k=new g(a,i.name.toLowerCase(),i.size);a.addCommand(i.name.toLowerCase(),k)}var j=a.config.minFontSize;a.ui.add("FontSize",CKEDITOR.UI_MENUBUTTON,{label:a.lang.fontsize.label,
```

Comment:
we need the original source, in additional to the compiled version

Reply:
the compiled version is the only source I've found, is there other place I need to change? checked previous MR and this is the only changed file as well, like https://github.com/strikingly/ckeditor/pull/2/commits/41fffd0f5bf0998a59a66064d8b4ed2b85bde15c